mod/servers.php

```diff
@@ -4,17 +4,17 @@
 
 require_once 'include/site-health.php';
 
-function servers_content(&$a) {
-
+function servers_content(&$a)
+{
 	$sites = array();
 
 	//Find the user count per site.
 	$r = q("SELECT `homepage` FROM `profile`");
-	if(count($r)) {
-		foreach($r as $rr) {
+	if (count($r)) {
+		foreach ($r as $rr) {
 			$site = parse_site_from_url($rr['homepage']);
-			if($site) {
-				if(!isset($sites[$site]))
+			if ($site) {
+				if (!isset($sites[$site]))
```

Comment:
@tobiasd they could be added - but their missing doesn't provoke a syntax error.